### ISSUE TITLE
Internal error: fixture integrity at browser - Playwright Tests

### ISSUE THREAD

**Issue by renhyl** [actions/runner-images#9516]
### Description

Issue with GitHub Actions failing

had changes done on separate branch before merging into my main branch., even though GA passed on my branch before the merge to main, after the merge I am perpetually getting `Internal error: fixture integrity at browser` when running Playwright tests.

![image](https://github.com/actions/runner-images/assets/2118334/c7773654-2d23-4726-8466-6f9d628dc142)

### Platforms affected

- [ ] Azure DevOps
- [X] GitHub Actions - Standard Runners
- [ ] GitHub Actions - Larger Runners

### Runner images affected

- [ ] Ubuntu 20.04
- [ ] Ubuntu 22.04
- [ ] macOS 11
- [ ] macOS 12
- [ ] macOS 13
- [ ] macOS 13 Arm64
- [ ] macOS 14
- [ ] macOS 14 Arm64
- [ ] Windows Server 2019
- [ ] Windows Server 2022

### Image version and build link

https://productionresultssa2.blob.core.windows.net/actions-results/fee10640-84b6-4d32-9013-48f40da40031/workflow-job-run-5264e576-3c6f-51f6-f055-fab409685f20/logs/job/job-logs.txt?rsct=text%2Fplain&se=2024-03-15T12%3A51%3A02Z&sig=YPtRCwQOXp%2FJjOJAOLXXdVzMmZ3YPuFr5VO4zRQo9t4%3D&sp=r&spr=https&sr=b&st=2024-03-15T12%3A40%3A57Z&sv=2021-12-02

### Is it regression?

no, i don't think so

### Expected behavior

I should not see internal error, test should run and either fail or pass without internal error.

### Actual behavior

Internal error hangs the Playwright tests.

### Repro steps

My Actions file:

name: Playwright Tests
on:
    push:
        branches: [main, master]
    pull_request:
        branches: [main, master]

env:
    APP_NAME: aaa
    APP_ENV: local
    APP_URL: http://localhost
    APP_SECRET: 'app_secret'
    APP_PORT: 5173

    API_PORT: 3000

    DB_HOST: 127.0.0.1
    DB_PORT: 3306
    DB_DATABASE: aaaa
    DB_USERNAME: root
    DB_PASSWORD: root

    WWWUSER: 1000
    WWWGROUP: 1000

    MAIL_MAILER: smtp
    MAIL_HOST: smtp.mailtrap.io
    MAIL_PORT: 2525
    MAIL_USERNAME: null
    MAIL_PASSWORD: null
    MAIL_ENCRYPTION: null
    MAIL_FROM_ADDRESS: null
    MAIL_FROM_NAME: '${APP_NAME}'

    JWT_SECRET: 'jwt_secret'

jobs:
    test:
        timeout-minutes: 60
        runs-on: ubuntu-latest
        steps:
            - name: Checkout code
              uses: actions/checkout@v4

            - name: Start MySQL
              run: |
                  sudo /etc/init.d/mysql start
                  mysql -u$DB_USERNAME -p$DB_PASSWORD -e "CREATE DATABASE IF NOT EXISTS $DB_DATABASE;"
            - uses: oven-sh/setup-bun@v1
              with:
                  bun-version: 'latest'
            - name: Install dependencies
              run: bun i
            - name: Setup database
              run: bun run db:migrate
            - name: Install Playwright Browsers
              run: bun playwright install --with-deps chromium
            - name: Run Playwright tests
              run: bun playwright test
            - uses: actions/upload-artifact@v4
              if: always()
              with:
                  name: playwright-report
                  path: playwright-report/
                  retention-days: 30

**Comment by MaksimZhukov** [comment]
Hello @renhyl!
Unfortunately, the link you provided does not work.
Could you please provide us with a link to the public repository where the issue is occurring?

**Comment by renhyl** [comment]
hi @MaksimZhukov , it is a private repo, do you think there is a way for us to figure out what went wrong?

I have been experimenting a little bit, and all I can think of is that when GitHub merged my feature branch into main, something didn't go quite right.
In process of me getting a grip on the issue I rebased my main back to the merge commit with feature branch just to find the actions run ok again.. 

to recap I had 2 branches
auth - feature branch, did all the work decided to raise PR for it and clicked "squash and merge" target: main
main - branch

main branch fire GA and it failed, I attempted to re-run few times each time fail with Internal error.

I made few commits attempting to change config/tests to see if I can fix it, failed.

Decided to rollback to when auth got merged and GA runs fine now..

**Comment by MaksimZhukov** [comment]
Unfortunately, we can't do anything without repro steps. Based on the description provided, it seems that the issue was caused by recent changes to the source code and is not related to images. 
I am closing the issue. Please contact us if you can provide us with repro steps or link to the public repo and we will reopen the issue.

**Comment by awhisler** [comment]
This is also happening to me. Ill create a new ticket

**Comment by sinasun** [comment]
I can confirm that this bug exists. Playwright tests work on branches, but when merging to main they fail. Tried downgrading, but still same issue.

**Comment by marcusNumminen** [comment]
I see this in our pipeline runs. The only code change I have made is that I structured the test with test steps....

**Comment by marcusNumminen** [comment]
Just realized that I also updated to playwright 1 45.3
